CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+## dev
+
+* `aframe-core` merged with `aframe`. `window.AFRAME` exposes what `aframe-core` was
+exposing previously (e.g., `AFRAME.registerComponent` vs.
+`AFRAME.aframeCore.registerComponent`).
+* Improved Android support.
+* Entities, including the scene, wait for their children to load before emitting the `loaded`
+event.
+* Scene `canvas` elements are now appended to the scene by default rather than
+to the document body. The scene can specify the canvas to render to.
+* Component property types. Property types define how a component property
```

Comment:
yeah, for folks who don't have hard wrapping, when you edit the file you have to manually tweak things. it's kind of hairy IMO